dev/benchmarks/microbenchmarks/lib/foundation/all_elements_bench.dart

```diff
@@ -0,0 +1,90 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart';
+// ignore: implementation_imports
+import 'package:flutter_test/src/all_elements.dart';
+
+import '../common.dart';
+
+const int _kNumIters = 10000;
+
+Future<void> main() async {
+  assert(false, "Don't run benchmarks in checked mode! Use 'flutter run --release'.");
+
+  List<Widget> _generateStacks(int count) {
+    print('Doing count $count');
+    if (count == 0) {
+      return const <Widget>[ SizedBox.shrink() ];
+    }
+    return List<Widget>.filled(
+      count,
+      Stack(children: _generateStacks(count - 1)),
+    );
+  }
+  runApp(MaterialApp(
+    home: Scaffold(
+      body: GridView.count(
+        crossAxisCount: 5,
+        children: List<Widget>.generate(100, (int index) {
+          return Center(
+            child: Scaffold(
+              appBar: AppBar(
+                title: Text('App $index'),
+                actions: const <Widget>[
+                  Icon(Icons.help),
+                  Icon(Icons.add),
+                  Icon(Icons.ac_unit),
+                ],
+              ),
+              body: Column(
+                children: const <Widget>[
+                  Text('Item 1'),
+                  Text('Item 2'),
+                  Text('Item 3'),
+                  Text('Item 4'),
+                ],
+              ),
+            ),
+          );
+        }),
+      ),
+    ),
+  ));
+
+  await SchedulerBinding.instance.endOfFrame;
+
+  final Stopwatch watch = Stopwatch();
+
+  print('flutter_test allElements benchmark... (${WidgetsBinding.instance.renderViewElement})');
+  // Make sure we get enough elements to process for consistent benchmark runs
+  int elementCount = collectAllElementsFrom(WidgetsBinding.instance.renderViewElement, skipOffstage: false).length;
+  while (elementCount < 6242) {
```

Comment:
magic number?

Reply:
This is the number of elements expected once the widget building has settled. What I found is that on some runs, I'd get to this point and only have <100 elements ready. This number was consistently the top number.